examples/extended/parallel/MPI/source/include/G4MPImanager.hh

```diff
@@ -153,11 +157,12 @@ class G4MPImanager
     G4bool is_slave_;
     G4bool is_extra_worker_;
     G4int rank_;
+    G4int provided_;
```

Comment:
```suggestion
```

I think this can now be removed as the latest commits use function local variables instead.